readthedocs/builds/managers.py

```diff
@@ -85,6 +86,60 @@ def get_object_or_log(self, **kwargs):
             log.warning('Version not found for given kwargs. %s' % kwargs)
 
 
+class VersionManagerBase(VersionManagerBaseClass):
```

Comment:
if we add it in the `Base` class then all the other classes that are inheriting (ie: `InternalVersionManagerBase`) from it will also show the message.